### ISSUE TITLE
randomBetween(int, int) and randomBetween(long, long) are broken

### ISSUE THREAD

**Issue by elnur** [Devskiller/jfairy#74]
The following invocations will always return 0:

```java
randomInt(Integer.MAX_VALUE)
randomBetween(0, Integer.MAX_VALUE)
```

The following invocation will always return a negative number: 

```java
randomBetween(0L, Long.MAX_VALUE)
```

The reason for this behavior for integers is this weird logic:

```java
int range = max - min + 1;
int randomInt = range > 0 ? this.random.nextInt(range) : 0;
return min + randomInt;
```

Subtracting 0 from the max integer value and then adding 1 to it overflows it and hence range is never above 0.

Something similar is being done for longs as well.

A quickfix is to subtract 1 from the max values but that's a huge WTF moment. These methods fail to deliver their promises.

**Comment by OlgaMaciaszek** [comment]
Thanks @elnur . We will provide a fix.

**Comment by mariuszs** [comment]
@OlgaMaciaszek you are so fast :)

**Comment by mariuszs** [comment]
Fixed in 527ac4957bad8e26071d8a0ea014993a9e085ce5
